### PR TITLE
11349 - Move Footer to bottom of page

### PR DESCRIPTION
**Pull request by LuisOlivoC**
Acceptance criteria

- [x] The footer on all pages needs to appear on the bottom of the page

- [x] Add 10px padding from the bottom

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -49,16 +49,21 @@ p.lead {
 }
 
 body {
+  overflow-x: hidden;
```
Comment (faustino-garcia) [inline app/assets/stylesheets/custom.css.scss:52]: Good job 👍, now the pages don't show the extra space on the right

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -49,16 +49,21 @@ p.lead {
...
+  min-height: 100vh;
+  padding-bottom: 70px;
 }
 
 footer {
+  position: absolute;
+  bottom: 0;
   margin-top: 45px;
-  padding-top: 5px;
+  padding: 5px 10px 10px;
```
Comment (faustino-garcia) [inline app/assets/stylesheets/custom.css.scss:63]: Nice job reducing the code by using only one pading property instead of three.

**Review by faustino-garcia — APPROVED** [review]
Well done Luis, in my opinion the changes were made correctly.

**Review by coderdave — APPROVED** [review]

**Review by josemorin — APPROVED** [review]

**Review by cdelacruz8512 — APPROVED** [review]
Looks good, I tested it and I approve it

**Pull request merged**